src/api/objects/ObjectAPI.js

```diff
@@ -312,6 +319,35 @@ define([
         });
     };
 
+    /**
+     * Register a new object interceptor.
```

Comment:
Let's add a little more documentation here to explain what get interceptors are for.

Reply:
Done